spec/jobs/irs_attempts_events_batch_job_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe IrsAttemptsEventsBatchJob, type: :job do
+  describe '#perform' do
+    events = { key1: 'SomeEncryptedEvent1' }
+    Result = Struct.new(:filename, :iv, :encrypted_key, :encrypted_data, keyword_init: true)
+
+    let(:redis_client) { instance_double(IrsAttemptsApi::RedisClient) }
+    let(:envelope_encryptor) { instance_double(IrsAttemptsApi::EnvelopeEncryptor) }
```

Comment:
let's not stub either of these, we can rely on the real behavior

Reply:
@zachmargolis Are you suggesting we should add some events to Redis in the test? I had figured we could assume that was already tested and stub it out, but I don't feel strongly. It would rid us of a lot of stubbing.

Reply:
especially for a small amount of events, I think it's fine to write to redis in tests, it helps us test the whole thing and be more certain in all the code

Reply:
Updated to not stub these.